docs/medhelm.md

```diff
@@ -10,29 +10,31 @@ MedHELM extends the HELM framework to evaluate **large language models (LLMs) in
 
 ---
 
-## Requirements 
+## Requirements
 
-1. [Conda](https://www.anaconda.com/docs/getting-started/getting-started) (~30 min to intall)
+Before you install MedHELM, make sure your system meets the following requirements:
+
+1. [Conda](https://www.anaconda.com/docs/getting-started/getting-started) (~30 minutes to install)
   
     Used to manage the Python virtual environment for MedHELM.
 
-2. [Google Cloud CLI](https://cloud.google.com/sdk/docs/install) (~30 min to install)
+2. [Google Cloud CLI](https://cloud.google.com/sdk/docs/install) (~30 minutes to install)
 
-    Required for downloading MedHELM results stored on Google Cloud.
+    Required for downloading MedHELM results stored in Google Cloud.
 
-    **Note:** During installation, the installer will recommend Python 3.12 for full feature support. These features are not needed for MedHELM. We recommend using Python 3.10 for best compatibility with MedHELM.
+    **Note:** During installation, the installer will recommend Python 3.12 for full feature support. These features are not needed. MedHELM is **only** compatible with `Python 3.10`.
```

Comment:
No action needed: note that Python 3.10 reaches end of life in October 2026 i.e. 12 months from now.